lib/src/common/variables.js

```diff
@@ -455,19 +455,16 @@ export const componentTokens = {
     closeIconHeight: "24px",
     closeIconTopPosition: "20px",
     closeIconRightPosition: "20px",
-    closeIconBackgroundColor: "none",
+    closeIconBackgroundColor: globalTokens.hal_white,
```

Comment:
The value of this token does not match the values updated in the documentation.

Reply:
Fixed!